core-library/src/main/java/org/silverpeas/core/io/media/image/imagemagick/Im4javaImageTool.java

```diff
@@ -25,16 +25,28 @@
 
 import org.im4java.core.ConvertCmd;
 import org.im4java.core.IMOperation;
+import org.im4java.core.IdentifyCmd;
+import org.im4java.process.ArrayListOutputConsumer;
 import org.silverpeas.core.io.media.image.AbstractImageTool;
+import org.silverpeas.core.io.media.image.ImageInfoType;
 import org.silverpeas.core.io.media.image.ImageToolDirective;
 import org.silverpeas.core.io.media.image.option.AbstractImageToolOption;
+import org.silverpeas.core.io.media.image.option.AnchoringPosition;
 import org.silverpeas.core.io.media.image.option.BackgroundOption;
 import org.silverpeas.core.io.media.image.option.DimensionOption;
+import org.silverpeas.core.io.media.image.option.TransparencyColorOption;
+import org.silverpeas.core.io.media.image.option.WatermarkTextOption;
 
 import javax.inject.Singleton;
 import java.io.File;
+import java.util.Arrays;
 import java.util.Map;
 import java.util.Set;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.stream.Collectors'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)